src/map/packets/message_standard.h

```diff
@@ -28,6 +28,25 @@
 
 #include "basic.h"
 
+/*
+ * Valid MessageIDs
+ * Complete list at 1/27/76.DAT
+ */
+enum MSGSTD_ID : uint16
```

Comment:
```
enum class MsgStd
{
    Wait_Longer = 38;
    ...
```

Reply:
wait what, we're changing enum style again?

Reply:
This contradicts `MSGBASIC` code at https://github.com/DarkstarProject/darkstar/blob/842fa2d500940bdc5abdf454db470fb0ee8ee0cd/src/map/packets/message_basic.h - Are you sure you want this to become inconsistent?

Reply:
Yes, because of the implicit conversions in enums, I would prefer it to be an enum class.  It means you can't accidentally use a MSGBASIC enum in a function that wants a MSGSTD.  At some point we can update MSGBASIC to be the same, for the same reason

I do like consistency, but sometimes we'll make an exception to make it easier in a different way (runtime vs compiletime, really)

Reply:
i was referring to `Wait_Longer` instead of our usual all caps for enums `WAIT_LONGER`

Reply:
So you want me to static cast every time I need to yank out the `uint16` from this?

Reply:
Sorta, we can add another constructor to CMessageStandard that takes a MsgStd instead of a uint16 that just converts it and then calls the other one, which should catch all of your uses.

Reply:
You seem to have some idea of how you want this to work, can you give me some code to work off of and I'll do it.